app/src/main/java/com/github/wnder/Score.java

```diff
@@ -18,16 +19,17 @@
      * @param distanceReference radius, "permitted error" in KM
      * @return score
      */
```

Comment:
Don't forget to change the doc (+ precise if radius needs to be in m or km)

Reply:
fixed